scanner/combiner.go

```diff
@@ -0,0 +1,54 @@
+package scanner
+
+import (
+	"context"
+
+	"github.com/brimsec/zq/ast"
+	"github.com/brimsec/zq/filter"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zng"
+)
+
+type combinerScanner struct {
+	combiner zbuf.Reader
+	scanners []Scanner
+}
+
+// NewCombinerScanner returns a Scanner that combines the records scanned from
+// a set of filtered readers.
+func NewCombinerScanner(ctx context.Context, readers []zbuf.Reader, spans []nano.Span, cmp zbuf.RecordCmpFn, f filter.Filter, filterExpr ast.BooleanExpr) (Scanner, error) {
```

Comment:
Nit: This imports as scanner.NewCombinerScanner, so including Scanner in the name is redundant. How about just calling it Combiner (so it imports as scanner.Combiner)?